### DISCUSSION TITLE
Dúvida sobre CodMun Pagador

### DISCUSSION THREAD

**Discussion by LeoBOrtigoza-zz** [bacen/pix-api#389] [Q&A]
Olá @ninrod, tudo bem?

Qual é a regra para identificar o código do município do pagador?

**Comment by ninrod** [comment]
boa noite @LeoBOrtigoza, 

Funciona assim (doc. do parâmetro `codMun` do endpoint `GET /cobv/{pixUrlAccessToken}`:

![image](https://user-images.githubusercontent.com/8352747/118053296-a1540a00-b35a-11eb-9262-334c0a955c41.png)


[Códigos dos Municípios - IBGE](https://www.ibge.gov.br/explica/codigos-dos-municipios.php)

**Reply by LeoBOrtigoza-zz** [reply]
bom dia @ninrod!

sim sim, isso tranquilo. Acho que não fui claro, quando eu digo a regra, é mais para saber como eu sei a cidade da pessoa? 
Por exemplo: pela localização que está lendo o pagamento, pelo endereço credenciado na conta bancária, pelo último endereço usado, ou até mesmo o endereço fiscal.

**Reply by ninrod** [reply]
o `codMun`, que é uma informação do PSP do pagador, refere-se ao domicílio bancário do usuário pagador.

**Reply by LeoBOrtigoza-zz** [reply]
Muito Obrigado! Perfeito, era mais para confirmar.
